### PR TITLE
executor: support Chunk for AnalyzeExec

### PR DESCRIPTION
**Pull request by zz-jason**
to #5261

### CODE REVIEW

File: executor/builder.go [hunk 1]
```diff
@@ -964,27 +964,36 @@ func (b *executorBuilder) buildAnalyzeColumnsPushdown(task plan.AnalyzeColumnsTa
...
 	}
-	b.err = setPBColumnsDefaultValue(b.ctx, e.analyzePB.ColReq.ColumnsInfo, cols)
+	b.err = errors.Trace(setPBColumnsDefaultValue(b.ctx, e.analyzePB.ColReq.ColumnsInfo, cols))
 	return e
 }
 
 func (b *executorBuilder) buildAnalyze(v *plan.Analyze) Executor {
 	e := &AnalyzeExec{
-		baseExecutor: newBaseExecutor(nil, b.ctx),
+		baseExecutor: newBaseExecutor(v.Schema(), b.ctx),
```
Comment (shenli) [inline executor/builder.go]: This is a bug-fix?

Comment (zz-jason) [reply executor/builder.go]: It's not. I think it's better to not let `e.Schema()` returns a nil pointer, this can help us to avoid some potential risk.

**Comment by shenli** [comment]
LGTM

**Pull request merged**